### ISSUE TITLE
Account types page incorrectly say you have 512Mb of free memory on CF apps

### ISSUE THREAD

**Issue by knolleary** [ibm-cloud-docs/account#5]
1. What are the details of the issue you're experiencing?

The documentation on [account types](https://cloud.ibm.com/docs/account?topic=account-accounts) says the Lite account gets 256Mb of memory, PAYG and Subscription gets 512Mb of memory.
![image](https://user-images.githubusercontent.com/51083/80116160-865efe00-857d-11ea-941f-52d6fe05fbc4.png)

Later in the text it says:

> With a Pay-As-You-Go account, you can access the full IBM Cloud catalog, including all free plans, and you get double the free runtime memory at 512 MB per month. 

This is no longer correct.

Following complaints from users that they were suddenly getting billed for apps that had been running for free for some time, I have discovered this blog post from December: https://www.ibm.com/cloud/blog/announcements/plan-updates-for-ibm-cloud-foundry-public

In summary, the Lite account still gets 256Mb of free memory.

The PAYG/Sub accounts get:
 - 256Mb of memory for *some* runtimes
 - Absolutely no free memory for Java, Node, ASP, Swift.

It is disappointing that no communication was sent to users to notify them they were about to incur costs that had not previously been there - but I realise this issue isn't the place to resolve that problem.

2. Is this issue in reference to a specific page? If so, please provide a link.

https://cloud.ibm.com/docs/account?topic=account-accounts

3. Do you have a suggestion about how to resolve the issue?

Fix the information on the table so our users don't think they are getting something for free and then get an unexpected bill at the end of the month.

**Comment by hlucey** [comment]
The table was updated to reflect the correct information https://cloud.ibm.com/docs/account?topic=account-accounts.
